resume.md

```diff
@@ -1,49 +1,49 @@
-* ...
+**Undergraduate Researcher, Software/Hardware**: Embodied Dexterity Group| Berkeley, CA ... *November 2019 - Present*
+
+* Led **team of 3** in designing a physics engine and GUI in Python to interface with a prosthetic arm, simulating the arm's movement and returning touch-related feedback. Incorporated the Model/View/Controller (MVC) design pattern to organize physics, GUI, and communication methods.
+* Established TCP connection to LabVIEW program using Python's socket library. Prevented packet loss and out-of-order delivery during communication, ensuring that any movement with our prosthetic arm resulted in accurate simulation and feedback. Allowed for meaningful, confident observations from the collected data.
+* Utilized multithreading to run communication/physics and GUI processing in parallel, overcoming stutters in the GUI refresh rate. Enabled the GUI to run at a consistent 60 frames per second, allowing test subjects to focus more easily on the experiment and preventing a potential study limitation.
+
+**Lead Machinist**: FIRST Robotics Competition Team 687 | Carson, CA ... *June 2015 - May 2018*
+
+* Led machining operations for a **10 member team**. Trained a total of **20+ students** on G-Code programming and proper manufacturing techniques to help them grow as engineers.
+* Improved record-keeping by implementing a detailed digital logbook of all completed and pending operations to keep team members updated on goals and progress while speeding up production times.
 
 ## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
-
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
+
+**Heal-Bot**: *The Quarantine Companion & Aide* | heal-bot.netlify.app ... *June 2020*
+
+* Developed Heal-Bot in order to provide pandemic information and support, tackling many of the lifestyle changes for those in quarantine. Received feedback from **10+ users**. 
+* Worked on Full Stack development to deploy Heal-Bot to the web using Netlify and Heroku.
+* Planning on integrating analytical tools to monitor monthly user activity and effectiveness of natural language processing.
```

Comment:
Be specific - any specific commands you've embedded into your project?  What kind of operations can this bot do?